reference/constraints/_payload-option.rst.inc

```diff
@@ -0,0 +1,6 @@
+payload
+~~~~~~~
+
+**type**: ``mixed`` **default**: ``null``
+
+This option can be used to attach arbitrary domain-specific data to a constraint.
```

Comment:
shouldn't we provide the information how to access this payload and why this is can be useful? or is this a topic for a cookbook entry?